modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -12,11 +12,16 @@
  * details.
  */
 
+import './DocumentLibrary.scss';
+
 import ClayButton, {ClayButtonWithIcon} from '@clayui/button';
 import ClayCard from '@clayui/card';
 import {ClayInput} from '@clayui/form';
 import ClayIcon from '@clayui/icon';
+import ClayProgressBar from '@clayui/progress-bar';
+import axios from 'axios';
```

Comment:
@alinedoleron, can we add the import following the alphabetic order? This should be on line 17, except if SF tells us to not do it.

Reply:
I suspect this but I thought SF would fix it. I'll make the change

Reply:
Ops, now when I tried the line got red and I used the the Fix auto-fixable problems tools I it replaced to the previous place.

Reply:
Let's keep where they are.